core/CMakeLists.txt

```diff
@@ -20,5 +20,3 @@ add_library(core
   CPUFactory.cpp
   CPUTopology.cpp
   )
-target_include_directories(core SYSTEM PRIVATE ${Boost_INCLUDE_DIRS})
-target_link_libraries(core PUBLIC SPARTA::sparta)
```

Comment:
Since I don't do a conda install of sparta, this fails for me: sparta  include paths and link paths are not passed properly.